static/js/controllers/messages-content.js

```diff
@@ -172,8 +172,19 @@ var _ = require('underscore'),
       };
 
       $scope.addRecipients = function(to) {
+        var realTo;
+        if (to.facility) {
+          realTo = to.facility;
+        } else if (to.contact) {
+          if (to.contact._id) {
+            realTo = to.contact;
+          } else if (to.contact.name) {
+            realTo = to.contact.name;
```

Comment:
How will this work? If we just have a name and no phone number, won't it fail validation? When will we have a name but no ID?